### PR TITLE
add requirements.txt

### PR DESCRIPTION
**Pull request by cap10morgan**
As only an occasional Python developer, I usually look for a requirements.txt file so I can just do `pip install -r requirements.txt` to get up and running. I know the requirements for this are very light, but even then specifying compatible versions is useful.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+boto==2.38.0
```
Comment (bchew) [inline requirements.txt]: Perhaps ``>=`` instead? (just thinking for when new versions of boto get released, although the boto version requirement is much lower).

**Comment by bchew** [comment]
@cap10morgan thanks for the PR! Just a minor comment, let me know what you think and I would merge it in if you find that acceptable. :)

File: requirements.txt (hunk 1 above)
Comment (cap10morgan) [inline requirements.txt]: Sure, I'm happy to leave the version constraint details up to you. You know better than I do what is and is likely to remain compatible.

Comment (bchew) [inline requirements.txt]: Cool, could you make the change and push it, I'll merge it in after, thanks.

**Comment by cap10morgan** [comment]
@bchew OK, that's done. Sorry for the delay.

**Comment by bchew** [comment]
@cap10morgan thanks!

**Pull request merged**